README.md

```diff
@@ -1,134 +1,145 @@
 
-The HTML element is built on top of the [Web Components specifications](https://www.webcomponents.org/introduction) and powered by the [Polymer library](https://www.polymer-project.org/). Although, there is no additional knowledge about how Polymer works necessary to use the console.
+The HTML element is built on top of the [Web Components specifications](https://www.webcomponents.org/introduction) and powered by the [Polymer library](https://www.polymer-project.org/). Familiarity with Polymer isn't necessary to use the console.
 
-The following sections give you a brief overview on how to build and use the console. For a much more in depth overview, please see the [docs](docs) directory in this repository. 
+The following sections briefly describe how to build and use the console. For more information, see the [docs](docs) directory in this repository. 
 
 ## Using the API console
 
-The API Console comes in two flavors which you can choose from: as a **standalone web-application** and as an embeddable **HTML element**. 
+API Console comes in two flavors.
+
+* A **standalone web-application**  
+* embeddable **HTML element**  
+
+You can select which one suits your needs.
 
 ### Run as a standalone web-application
 
-API Console as a standalone application should be used to display the documentation for your API as a web page. The application supports [Deep linking][deep linking] which allows you to share a link to a particular part of you API documentation. You can find a basic example of the standalone application on our [demo application] web page.
+Use API Console as a standalone application to display the documentation for your API as a web page. The application supports [Deep linking][deep linking], which allows you to share a link to a particular part of your API documentation. You can find a basic example of the standalone application on our [demo application] web page.
 
 To build the API Console as a standalone application use one of our [build tools].
 
 ### Embed as an HTML element
 
-The API Console was build on top of new Web Components specification. When you include sources of the console into your web application it will register a new HTML element, the `<api-console>` element, that can be used as any other element on the page or web application. This means that you can embed the console into your blog post or as a part of a press release, and your users will be able to get know your API without being redirected to another web page.
+The API Console was built on top of a new Web Components specification. When you include sources of the console into your web application it registers a new HTML element, `<api-console>`. You can use this element in the same way as any other element on the page or web application. For example, you can embed the console into your blog post or as a part of a press release. Your users can explore your API without being redirected to another web page.
```

Comment:
Instead of `on top of a new Web Components specification` it should be `on top of the new Web Components specification`, right?